style.css

```diff
@@ -33,11 +67,31 @@
   cursor: pointer;
 }
 
+.hide {
+  display: none;
+}
+
+.closeBtn {
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+}
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=that%20you%20chose.-,You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins),-using%20the%20templates), Please add the correct margin for this `close button icon` because its suppose to be up a little bit to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/223230684-89d0bd13-099d-447d-8d86-63c88ca927c5.png)  | ![image](https://user-images.githubusercontent.com/68796232/223230663-a6ef44fb-63b0-4de6-b8af-3e6629a997d0.png)  |

Reply:
Thank you for your feedback
I shall make these changes and again ask for a review